ganga/GangaLHCb/Lib/RTHandlers/RTHUtils.py

```diff
@@ -1,6 +1,5 @@
 #\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\#
 import os
-import tempfile
```

Comment:
I assume this was an unused import and removing it won't cause something to break